.github/workflows/publish.yml

```diff
@@ -1,16 +1,23 @@
+name: Publish
```

Comment:
> Thanks! Even thought this will create two job listings until the logs expire

no problem! as noted above, you won't see two job listings anymore 🎉 - better handled by GitHub frontend now.

And yeah, I hate _not_ setting a workflow `name:` - looks ugly having workflow names based on the yml filename/path.

Reply:
![Screen Shot 2022-05-26 at 10 06 32 am](https://user-images.githubusercontent.com/1818757/170388971-350488c0-3027-4b84-ad4c-f5a2ee2b2a71.png)

Hey @jpogran just to circle back on this - as you can see the sidebar only shows the current mainline workflow names (not the legacy name in the case of publish here).

Also [viewing the Publish workflow runs](https://github.com/hashicorp/js-releases/actions/workflows/publish.yml) groups up all runs from `publish.yml` - regardless of it's workflow name at the time.